ONSdigital/ras-frontstage
# add GA event and google tag manager

# Motivation and Context

Adding google tag manager to allow GA admin to do their magic without needing to ask developers

# What has changed



Added an event to be triggered for EQ survey launch
Also google tag manager scripts

# How to test?



You could create a google analytics account and update the env vars for ga id and see the events are getting triggered. Or just run the tests and make sure i've not broken anything. 

# Links



https://trello.com/c/M6sK8wZ4/90-extend-eq-google-analytics-tracking-code-to-include-ras-sm-for-cross-domain-tracking

# Screenshots (if appropriate):
